src/BuiltInTools/dotnet-watch/Process/RunningProject.cs

```diff
@@ -81,16 +82,34 @@ public async ValueTask<bool> WaitForProcessRunningAsync(CancellationToken cancel
             }
         }
 
-        public async Task<int> TerminateAsync(bool isRestarting)
+        /// <summary>
+        /// Terminates the process if it hasn't terminated yet.
+        /// </summary>
+        public async Task TerminateAsync()
         {
-            IsRestarting = isRestarting;
-
             if (!_isDisposed)
             {
                 processTerminationSource.Cancel();
             }
 
-            return await RunningProcess;
+            _ = await RunningProcess;
```

Comment:
Can we just return the task here instead of awaiting it? If not, should a comment be added to explain why?